sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -324,6 +324,18 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
     val webpackMonitoredFiles: TaskKey[Seq[File]] =
       taskKey[Seq[File]]("Files that trigger webpack launch")
 
+    /**
+      * Additional arguments to webpack
+      *
+      * Defaults to an empty list.
+      *
+      * @group settings
+      */
+    val webpackExtraArgs = SettingKey[Seq[String]](
```

Comment:
This key already exists in master. So, you should rebase your work on top of master.

In the end, it’s probably simpler to start a fresh PR on top of master.